hmri_proc_MPMsmooth.m

```diff
@@ -84,16 +89,22 @@
 end
 
 % Flags for image calculation
-ic_flag = struct(...
-    'dtype', 16, ... % keep things in floats
+ic_flag = struct(...%    'dtype', 16, ... % keep things in floats
```

Comment:
What does this do? Does it just maintain the input data type?

Reply:
Since most (all?) the maps used are encoded in a float format, setting the `dtype` flag to 16 forces `spm_imcalc` to write out the resulting image in float format as well. Otherwise the default format will be in `int16`, see [here](https://github.com/spm/spm12/blob/master/spm_imcalc.m#L21). 

A more elegant way would be to check the format of the input images and to adjust the output format accordingly... Shall we add an issue about this?

Reply:
OK, so the commented out part in the new version means that `int16` will be used? Is this what you intended?